libp2p/standard_setup.nim

```diff
@@ -37,27 +36,33 @@ proc newStandardSwitch*(privKey = none(PrivateKey),
     transports = @[Transport(TcpTransport.init(transportFlags))]
     muxers = {MplexCodec: mplexProvider}.toTable
     identify = newIdentify(peerInfo)
-  when libp2p_secure == "noise":
-    let secureManagers = {NoiseCodec: newNoise(seckey).Secure}.toTable
-  else:
-    let secureManagers = {SecioCodec: newSecio(seckey).Secure}.toTable
+
+  var 
+    secureManagerInstances: OrderedTable[string, Secure]
```

Comment:
meh - just use a `seq` - ordering by protocol name just arbitrarily happens to work but that's not the intent of the change

Reply:
You mean refactor also `newSwitch` signature to use `seq` ? 
Internally switch deep still has and needs a table tho or refactor the whole thing? 
Tbh we don't need tables...

Reply:
not sure - the point is to establish the order using a seq - how deep down that should go depends..

Reply:
Yeah, I agree `seq` is better here, you only have to change `switch` because `mutistreamselect` already uses a `seq`.  Should be fairly simple, just replacing the table of `secure` managers for a `seq`.